src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -333,38 +335,32 @@ private void EventLoop()
                     Debug.Assert(numEvents > 0, $"Unexpected numEvents: {numEvents}");
 
                     bool enqueuedEvent = false;
-                    for (int i = 0; i < numEvents; i++)
+                    foreach (var socketEvent in new ReadOnlySpan<Interop.Sys.SocketEvent>(buffer, numEvents))
```

Comment:
This change was made just for readability / safety?  I'm not seeing how this would have a perf benefit.

Reply:
> This change was made just for readability / safety?

Yes, exactly. No perf benefit, just more `Span` and less pointers ;)